gallery/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sonatype/react-shared-components-gallery",
-  "version": "10.2.6",
+  "version": "10.2.8",
```

Comment:
Why is it bumped two versions?

Reply:
Main has been updated since I began the ticket, so I was updating to the newest version according to main (which is currently 10.2.7)